File: Java/1092.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int a = sc.nextInt();
		int b = sc.nextInt();
		int c = sc.nextInt();
		
		long count = 1;
		
		while(count%a!=0 || count%b!=0 || count%c!=0) count++;
		
		System.out.println(count);
	}
}

File: C++/1092.cpp
#include <iostream>

using namespace std;

int main(){
	int a, b, c;
	cin >> a >> b >> c;
	
	long int count = 1;
	
	while(count%a!=0 || count%b!=0 || count%c!=0) count++;
	
	cout << count;
}